dist/Makefile

```diff
@@ -49,13 +49,15 @@ install_obs_bin: system_dirs
 
 install_crontabs:
 	$(INSTALL) -m 644 cleanup_scm_cache.cron $(DESTDIR)/etc/cron.d/cleanup_scm_cache
+	$(INSTALL) -m 644 obs_api_delayed_jobs_monitor.cron $(DESTDIR)/etc/cron.m/obs_api_delayed_jobs_monitor
```

Comment:
What is cron.m?

Reply:
"cron.minutely", just like how the other dir is called cron.d for "cron.daily".

Reply:
Maybe I've got this wrong..